src/scss/components/person/_person-teasers.scss

```diff
@@ -44,9 +44,11 @@
 
       h2,
       h3 {
+        font-size: 1.4em;
```

Comment:
I think I see this is needed to override the style that it is now getting from general card styles.